frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -289,6 +290,7 @@ private void serveNewTableRowLinkedRelease(ResourceRequest request, ResourceResp
 
                 final String fullname = vendor != null ? vendor.getFullname() : "";
                 ReleaseLink linkedRelease = new ReleaseLink(release.getId(), fullname, release.getName(), release.getVersion());
+                linkedRelease.setReleaseRelationship(ReleaseRelationship.REFERRED);
```

Comment:
The ticket says that is-sub-project (=CONTAINED) should be the standard. For linked releases of releases, the ticket does not say anything. So just to check: is this really Ok here?